lua/smart-splits/config.lua

```diff
@@ -79,6 +79,16 @@ function M.set_default_multiplexer()
     return
   end
 
+  -- if running in a GUI instead of terminal TUI, disable mux
+  -- because you aren't in any terminal, you're in a Neovim GUI
+  local current_ui = vim.tbl_filter(function(ui)
+    return ui.chan == 1
+  end, vim.api.nvim_list_uis())[0]
+  if not current_ui.stdin_tty and not current_ui.stdout_tty then
```

Comment:
I would add a check for nil for `current_ui`, so 
```suggestion
  if current_ui and not current_ui.stdin_tty and not current_ui.stdout_tty then
```

Reply:
Good shout. I guess it could be `nil` if you're connecting remotely.